bundles/org.openhab.binding.solax/README.md

```diff
@@ -0,0 +1,210 @@
+| lastUpdateTime           | DateTime                   | Last time when a call has been made to the inverter                                      |
+| rawData                  | String                     | The raw data retrieved from inverter in JSON format. (Usable for channels not implemented. Can be consumed with the JSONpath transformation |
+
+### Properties
+
+| Property          | Description                               |
+|-------------------|-------------------------------------------|
+| serialWifi        | The wifi module's serial number           |
+| inverterType      | Inverter Type (for example X1_HYBRID_G4)  |
+
+
+## Full Example
+
+Here are some file based examples
+
+### Thing Configuration
+
+```java
+// The local connect bridge
+Bridge solax:localConnect:localBridge "Solax Inverter LocalBridge" [refresh=10, password="<SERIAL NUMBER OF THE WIFI>", hostname="<local IP/hostname in the network>"] {
+    Thing inverter MyInverter "MyInverter via LocalBridge" [serialWifi="<SERIAL NUMBER OF THE WIFI>", inverterType="X1_HYBRID_G4"]
+}
+```
+
+### Item Configuration
+
+```java
+Group gSolaxInverter "Solax Inverter" <energy> (boilerRoom)
+Group solarPanels "Solar panels" <energy> (gSolaxInverter)
+String solaxSerial "Serial Number [%s]" <energy> (gsolax_inverter) { channel="solax:inverter:localBridge:MyInverter:serialWifi" }
+
+Number solaxPowerWest "West [%.0f W]" <solarplant> (gsolax_inverter,EveryChangePersist,solarPanels){ channel="solax:inverter:localBridge:MyInverter:pv1Power" }
+Number solaxPowerEast "East [%.0f W]" <solarplant> (gsolax_inverter,EveryChangePersist,solarPanels){ channel="solax:inverter:localBridge:MyInverter:pv2Power" }
+Number solaxBatteryPower "Battery power [%.0f W]" <energy> (gsolax_inverter,EveryChangePersist) { channel="solax:inverter:localBridge:MyInverter:batteryPower" }
+Number solaxBatterySoc "Battery SoC [%.0f %%]" <batterylevel> (gsolax_inverter,EveryChangePersist) { channel="solax:inverter:localBridge:MyInverter:batteryStateOfCharge" }
+
+Number solaxFeedInPower "Feed-in power (CEZ) [%.0f W]" <energy> (gsolax_inverter,EveryChangePersist) { channel="solax:inverter:localBridge:MyInverter:feedInPower" }
+Number solaxAcPower "Invertor output power [%.0f W]" <energy> (gsolax_inverter,EveryChangePersist){ channel="solax:inverter:localBridge:MyInverter:inverterOutputPower" }
+
+String solaxInverterType "Inverter Type [%s]" <energy> (gsolax_inverter) { channel="solax:inverter:localBridge:MyInverter:inverterType"}
+String solaxUploadTime "Last update time [%s]" <calendar> (gsolax_inverter) { channel="solax:inverter:localBridge:MyInverter:lastUpdateTime" }
+String solaxRawData "Raw data [%s]" <data> (gsolax_inverter) { channel="solax:inverter:localBridge:MyInverter:rawData" }
+```
+
+### Sitemap Configuration
+
+```perl
+Frame label="Solar power strings" {
+                        Text item=solaxPowerEast valuecolor=[<=30="gray",<=300="red", <1500="orange", >=1500="green"] {
```

Comment:
Can you reduce the indentation to four space?

Reply:
wooh... took me a while to manually format this but I did it...

Reply:
It looks much better now! 👏